psetv2/updater.go

```diff
@@ -539,25 +485,29 @@ func (u *Updater) AddInReissuance(arg AddInReissuanceArgs) error {
 	)
 	rawAsset = append([]byte{0x01}, rawAsset...)
 	tokenAsset := elementsutil.AssetHashFromBytes(rawAsset)
-	addr, _ = address.FromConfidential(arg.TokenAddress)
+
+	tokenScript, tokenBlindingKey := parseAddress(arg.TokenAddress)
+
 	tokenOut := OutputArgs{
-		Asset:        tokenAsset,
-		Amount:       arg.TokenAmount,
-		Script:       addr.Script,
-		BlindingKey:  addr.BlindingKey,
-		BlinderIndex: uint32(inputIndex),
+		Asset:       tokenAsset,
+		Amount:      arg.TokenAmount,
+		Script:      tokenScript,
+		BlindingKey: tokenBlindingKey,
+	}
+	if tokenBlindingKey != nil {
+		tokenOut.BlinderIndex = uint32(inputIndex)
```

Comment:
not needed?

Reply:
golang is complaining, inputIndex is an `int` whereas blinderIndex is an `uint32`

Reply:
no i mean, we already take care of this [here](https://github.com/vulpemventures/go-elements/blob/78befafcb2f10aec4031aa10df580caad9c68ecd/psetv2/updater.go#L480-L482), therefore this is not needed.

Reply:
remove this if block and it's good to be merged